### PR TITLE
make it pretty

### PR DESCRIPTION
**Pull request by CamronStaley**
## What changes are proposed in this pull request?

Aligning the toast for the query performance panel more closely with the mocks.

## How is this patch tested? If it is not, please explain why.

Local dev:

image

### Is this a user-facing change that should be mentioned in the release notes?

-   [x] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.
### What areas of FiftyOne does this PR affect?

-   [x] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

## Summary by CodeRabbit

- **New Features**
	- Enhanced `QueryPerformanceToast` to display for a longer duration and show by default.
	- Updated theme colors for better responsiveness and styling.
	- Improved layout of `Toast` notifications for a cohesive user experience.

- **Bug Fixes**
	- Adjusted button text for clarity and consistency.

- **Chores**
	- Updated Material-UI imports to the latest version for improved performance and features.

### CODE REVIEW

File: app/packages/app/src/components/QueryPerformanceToast.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-import { Box, Button, Typography } from "@material-ui/core";
+import { Box, Button, Typography } from "@mui/material";
 import { Bolt } from "@mui/icons-material";
 import React, { useEffect, useState } from "react";
 import { createPortal } from "react-dom";
 import { atom, useRecoilState } from "recoil";
+import { useTheme } from "@fiftyone/components";
 
-const SHOWN_FOR = 10000;
+const SHOWN_FOR = 50000;
```
Comment (minhtuev) [inline app/packages/app/src/components/QueryPerformanceToast.tsx:11]: This might be 50 seconds instead of 5 seconds btw ~

**Review by minhtuev — APPROVED** [review]
Thanks Cam, looking great! Small comment about duration, otherwise LGTM ✅

File: app/packages/app/src/components/QueryPerformanceToast.tsx (hunk 1 above)
Comment (CamronStaley) [reply app/packages/app/src/components/QueryPerformanceToast.tsx:11]: I just copied this from your PR: https://github.com/voxel51/fiftyone/pull/4966. Do you wanna merge that one and I can rebase onto it or merge this one and do the opposite?

Comment (minhtuev) [reply app/packages/app/src/components/QueryPerformanceToast.tsx:11]: My bad, my PR was WIP so I set a high number for debugging; you can set to `5000` and merge directly.

**Pull request merged**